<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>영어 학습기</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabStudy1">
    <attribute name="title">
     <string>문장 학습(기본 방법)</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>102</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>저장된 문장 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSavedText">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>102</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>익숙한 문장 : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFriendlyText">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>102</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>어려운 문장 : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelDifficultText">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>98</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>학습한 문장 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelStudiedText">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>102</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>선택된 문장 : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSelectedText">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>학습 범위 선택</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>461</width>
         <height>18</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButtonAll">
          <property name="text">
           <string>모든 문장</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonStudied">
          <property name="text">
           <string>학습한 문장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonNotStudied">
          <property name="text">
           <string>학습안한 문장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonHard">
          <property name="text">
           <string>어려운 문장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonNew">
          <property name="text">
           <string>새로운 문장</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Study Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>651</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>한글 문장(문제)</string>
     </property>
     <widget class="QTextEdit" name="textEditQuestion">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>90</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>* 출제되는 문제는 중복되지 않습니다.</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>진행도</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarProgress">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelProgress">
         <property name="text">
          <string>0/0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>651</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>영어 문장 입력(정답)</string>
     </property>
     <widget class="QTextEdit" name="textEditAnswerInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditAnswer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>631</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>621</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="pushButtonAnswer">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>정답 확인</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNext">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>다음 문제</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRetry">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>다시 풀기</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>ADMUI3Lg</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>* 정답 확인은 Ctrl+Enter, 다음 문제 Ctrl+Right, 다시 풀기 Ctrl+Left</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>644</width>
        <height>17</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>반드시 들어가는 단어: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelContainWord">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>ADMUI3Lg</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabStudy2">
    <attribute name="title">
     <string>문장 학습(advanced)</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>368</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>필요한 기능을 고안중입니다. 아이디어가 있으면 메일로 보내주세요.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabEnter">
    <attribute name="title">
     <string>문장 입력</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>문장 입력</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 문장(한글)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditInputKorean">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>631</width>
        <height>81</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditInputEnglish">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>631</width>
        <height>81</height>
       </rect>
      </property>
      <property name="tabChangesFocus">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 문장(영어)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>중요 단어(단어, 뜻 순서로)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>76</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>저장된 문장 : </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSavedText2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>18</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>437</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>단어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditWord"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>뜻</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditWord2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>문장 저장</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>문장 수정</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>471</height>
      </rect>
     </property>
     <property name="title">
      <string>문장 수정</string>
     </property>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>491</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ADMUI3Lg</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>* 검색할 문장은 반드시 영어 문장만 입력해야하며, 앞의 일부분만 입력해도 됩니다.</string>
      </property>
     </widget>
     <widget class="QListView" name="listViewSearchResult">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>611</width>
        <height>61</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditSelectEnglish">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>611</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>선택한 문장(영어)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>선택한 문장(한글)</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditSelectKorean">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>611</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>90</width>
        <height>14</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>검색 결과 : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSearchCount">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>611</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>문장 입력</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditSearchKeyword"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSearch">
         <property name="text">
          <string>문장 검색</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>437</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>단어</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSelectWord">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>뜻</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSelectWord2">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonChange">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>문장 변경</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>문장 삭제</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSettings">
    <attribute name="title">
     <string>환경 설정</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>아직 미구현</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>198</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>건의 사항: mauver5445@gmail.com</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditSearchKeyword</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>listViewSearchResult</tabstop>
  <tabstop>textEditSelectEnglish</tabstop>
  <tabstop>textEditSelectKorean</tabstop>
  <tabstop>lineEditSelectWord</tabstop>
  <tabstop>lineEditSelectWord2</tabstop>
  <tabstop>pushButtonChange</tabstop>
  <tabstop>pushButtonDelete</tabstop>
  <tabstop>textEditInputKorean</tabstop>
  <tabstop>lineEditWord</tabstop>
  <tabstop>lineEditWord2</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButtonAnswer</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonRetry</tabstop>
  <tabstop>textEditQuestion</tabstop>
  <tabstop>textEditAnswerInput</tabstop>
  <tabstop>textEditAnswer</tabstop>
  <tabstop>pushButtonStart</tabstop>
  <tabstop>textEditInputEnglish</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
